lib/order.rb

```diff
@@ -0,0 +1,72 @@
+require "csv"
+require_relative "customer"
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products = {}, customer = nil, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+
+    if ![:pending, :paid, :processing, :shipped, :complete].include?(fulfillment_status)
+      raise ArgumentError.new("Invalid Status: #{fulfillment_status}. Must be one of the following: pending, paid, processing, shipped, or complete.")
+    end
+
+    @fulfillment_status = fulfillment_status
+  end
+
+  def total
+    sum = 0
+    total = 0
+
+    @products.each do |item, price|
+      sum += price
+    end
+
+    total = sum + sum * 0.075
+
+    return total.round(2)
+  end
+
+  def add_product(product_name, price)
+    if @products.keys.include?(product_name)
+      raise ArgumentError.new("{#{product_name} already added.")
+    end
+
+    @products[product_name] = price
+  end
+
+  def self.parser(row)
```

Comment:
Love that you broke this out into it's own helper method! 
Additionally, because it's only being called from within the class, it would be appropriate to make it a `private` method.